### PR TITLE
Bitfield: Add .natvis for easier debugging of BitFields in Visual Studio

### PR DESCRIPTION
**Pull request by phire**
### Before:
image

### After:
image

### CODE REVIEW

File: Source/Core/Common/Common.vcxproj.filters [hunk 1]
```diff
@@ -189,6 +189,8 @@
     <ClInclude Include="GL\GLInterfaceBase.h">
       <Filter>GL\GLInterface</Filter>
     </ClInclude>
+    <ClInclude Include="Assert.h" />
```
Comment (BhaaLseN) [inline Source/Core/Common/Common.vcxproj.filters]: Technically, they don't really belong to this PR, do they?

**Comment by BhaaLseN** [comment]
Nice stuff, didn't know this was possible. Sadface about VS not doing the Newline-at-EOF thingy tho.

File: Source/Core/Common/BitField.natvis [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      <Item Name="Size">$T2</Item>
+    </Expand>
+  </Type>
+
+
+  <!--Specialised versions for signed types-->
+  <Type Name="BitField&lt;*,*,__int64&gt;">
+    <!-- This is what I have do to get a sign extension in this crappy natvis "language"
+         Basically, we check the top bit, if it's one, we add the remaining
+         bits to the smallest (most negitive) number. -->
```
Comment (Alcaro) [inline Source/Core/Common/BitField.natvis]: Typo ('negitive').

File: Source/Core/Common/Common.vcxproj.filters (hunk 1 above)
Comment (phire) [inline Source/Core/Common/Common.vcxproj.filters]: No, visual studio automatically added them on save.

I see no reason to put them in a separate PR.
